src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,88 @@
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (currentPage: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const isSelectedFirstPage = currentPage === 1;
+  const isSelectedLastPage = currentPage === totalPages;
+
+  const handleSelectPrevPage = () => {
+    if (!isSelectedFirstPage) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleSelectNextPage = () => {
+    if (!isSelectedLastPage) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: isSelectedFirstPage,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={isSelectedFirstPage}
+          onClick={handleSelectPrevPage}
+        >
```

Comment:
Prevent the default action of the anchor tag when clicking it to avoid changing the URL. You can do this by calling `event.preventDefault()` in the `handleSelectPrevPage` function.